app/templates/orders/goods.hbs

```diff
@@ -1,63 +1,64 @@
 <div class="row my_orders">
-  <div class="small-12 columns content">
-    <div class="order-summary {{order.state}}">
-      {{partial 'orders/order_header'}}
-      {{partial 'orders/order_tabs'}}
-      <div class="tab-content">
-        <div class="goods-tab">
-	        {{#if hasRequestedGoods}}
-	        	{{!-- REQUESTS --}}
-	          <section class="details">
-	            <div class="section-title">
-	              {{ t "my_orders.details.goods_requested" }}
-	            </div>
+	<div class="small-12 columns content">
```

Comment:
Can we fix indentation in this file? it seems very odd.

Reply:
For all the non-critical changes, we can list them out and make another PR later. Not to delay the release :)